### PR TITLE
Quick view

### PR DESCRIPTION
**Pull request by matthaws**
Implements a proof of concept pop-out quickview on the contest show page. I used a new package (react-stickynode) to make the quickview sticky to the top of the screen as you scroll down, but we may eventually want to implement this ourselves for bragging rights.

### CODE REVIEW

**Review by Bhammy — COMMENTED** [review]
There's a switch quickview, but is there a switch out of quickview? Do we care if users can't switch out?

**Comment by LlanddewiLovesYou** [comment]
We talked.about that. There's tweaking to.be done for sure, and I think
they should def be able to close it.

File: src/components/entries/entryQuickView.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  if (entry) {
+    let country = countries[entry.country_id];
+    return (
+      <aside id="pop-out">
+        <div id="quick-view">
+          <Sticky>
+            <header>
+              <img src={country.flag_url} />
+              <p className="p--closeX" onClick={closeQuickView}>
+                X
```
Comment (Bhammy) [inline src/components/entries/entryQuickView.jsx]: Replace with String.fromCharCode(215) to get nice even 'x'.

**Review by Bhammy — CHANGES_REQUESTED** [review]

File: src/components/entries/entryQuickView.jsx (hunk 1 above)
Comment (matthaws) [reply src/components/entries/entryQuickView.jsx]: I didn't know that!

**Review by Bhammy — APPROVED** [review]

**Pull request merged**